tests/media.md

```diff
@@ -0,0 +1,36 @@
+# Notebooks for all media
+
+A collection of multimedia references to Notebooks for All.
+
+## Articles 
+
+* [Making Jupyter Notebooks More Accessible]
```

Comment:
I'm not sure which one this is and couldn't find it on my own. I'd rather have a link for this or remove it if we can't verify.

Reply:
Nevermind, it was at the bottom of the page. My mistake! 
```suggestion
* [Notebooks For All: Making Jupyter Notebooks More Accessible - Astrobites]( https://astrobites.org/2023/06/10/notebooks-for-all/)
```